pkg/securitycontextconstraints/sccmatching/provider.go

```diff
@@ -273,7 +275,7 @@ func (s *simpleProvider) ValidatePodSecurityContext(pod *api.Pod, fldPath *field
 				continue
 			}
 
-			if !allowedVolumes.Has(string(fsType)) {
+			if !allowsVolumeType(allowedVolumes, fsType, v.VolumeSource) {
```

Comment:
FSTypeToStringSetInternal above needs a `projected` case. Now it will error out.

Reply:
https://github.com/openshift/apiserver-library-go/blob/a2d020141c5f4c0bad6092323ea677011e679091/vendor/github.com/openshift/api/security/v1/types.go#L181
should be enough, no?